cdap-ui/cypress/support/commands.ts

```diff
@@ -57,3 +59,212 @@ Cypress.Commands.add('cleanup_pipelines', (headers, pipelineName) => {
       }
     });
 });
+
+Cypress.Commands.add(
+  'fill_GCS_connection_create_form',
+  (
+    connectionId,
+    projectId = DEFAULT_GCP_PROJECTID,
+    serviceAccountPath = DEFAULT_GCP_SERVICEACCOUNT_PATH
+  ) => {
+    cy.visit('/cdap/ns/default/connections');
+    cy.get('[data-cy="wrangler-add-connection-button"]', { timeout: 30000 }).click();
+    cy.get(`[data-cy="wrangler-connection-${ConnectionType.GCS}`).click();
+    cy.get(`[data-cy="wrangler-${ConnectionType.GCS}-connection-name"]`).type(connectionId);
+    cy.get(`[data-cy="wrangler-${ConnectionType.GCS}-connection-projectid"]`).type(projectId);
+    cy.get(`[data-cy="wrangler-${ConnectionType.GCS}-connection-serviceaccount-filepath"]`).type(
+      serviceAccountPath
+    );
+  }
+);
+
+Cypress.Commands.add('create_GCS_connection', (connectionId) => {
+  cy.fill_GCS_connection_create_form(connectionId);
+  cy.get(`[data-cy="wrangler-${ConnectionType.GCS}-add-connection-button"]`).click({
+    timeout: 60000,
+  });
+});
+
+Cypress.Commands.add('test_GCS_connection', (connectionId, projectId, serviceAccountPath) => {
+  cy.fill_GCS_connection_create_form(connectionId, projectId, serviceAccountPath);
+  cy.get(`[data-cy="wrangler-${ConnectionType.GCS}-test-connection-button"]`).click({
+    timeout: 60000,
+  });
+});
+
+Cypress.Commands.add(
+  'fill_BIGQUERY_connection_create_form',
+  (
+    connectionId,
+    projectId = DEFAULT_GCP_PROJECTID,
+    serviceAccountPath = DEFAULT_GCP_SERVICEACCOUNT_PATH
+  ) => {
+    cy.visit('/cdap/ns/default/connections');
+    cy.get('[data-cy="wrangler-add-connection-button"]', { timeout: 30000 }).click();
```

Comment:
it can take up to 30 seconds for adding a connection?

Reply:
So this is to find the add-connection button on the bottom left of connections. UI could potentially take 30 seconds to show the connections page for the first time when we just deployed and enabled dataprep app. This is to make sure the loading indicator is gone and we are able to click on the add connection button.